pkgs/applications/radio/aldo/default.nix

```diff
@@ -1,14 +1,19 @@
-{ stdenv, fetchurl, libao }:
+{ stdenv, fetchgit, libao, autoreconfHook }:
 
-stdenv.mkDerivation rec {
```

Comment:
Was there a particular reason for this change?

Reply:
I prefer to not have recursive attribute sets where variables can be borrowed and used anywhere, it makes the scope more confusing than a let block :)

Reply:
Ah, not a style I see often. Thanks for explaining!